lib/command/terraform/generate.rb

```diff
@@ -16,32 +16,20 @@
       WITH_INFO_HEADER = false
 
       def call
-        generate_common_configs
-        generate_app_configs
-      end
-
-      private
-
-      def generate_common_configs
-        cpln_provider = TerraformConfig::RequiredProvider.new(
-          "cpln",
-          source: "controlplane-com/cpln",
-          version: "~> 1.0"
-        )
-
-        File.write(terraform_dir.join("providers.tf"), cpln_provider.to_tf)
-      end
-
-      def generate_app_configs
         Array(config.app || config.apps.keys).each do |app|
-          config.instance_variable_set(:@app, app)
+          config.instance_variable_set(:@app, app.to_s)
           generate_app_config
         end
       end
 
+      private
+
       def generate_app_config
         terraform_app_dir = recreate_terraform_app_dir
 
+        generate_required_providers(terraform_app_dir)
+        generate_providers(terraform_app_dir)
```

Comment:
Added exceptions handling but rescuing from `StandardError` i/o `IOError`, can you review this file again?

Reply:
We are rescuing from `StandardError` i/o `IOError` because `TerraformConfig::RequiredProvider` or `TerraformConfig::Provider` can also raise exceptions